apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -734,6 +764,113 @@ describe('LIVECHAT - rooms', function () {
+
+			const newVisitor = await createVisitor(initialDepartment._id);
+			const newRoom = await createLivechatRoom(newVisitor.token);
+
+			await request.post(api('livechat/room.forward')).set(credentials).send({
+				roomId: newRoom._id,
+				departmentId: forwardToOfflineDepartment._id,
+				clientAction: true,
+				comment: 'test comment',
+			});
+
+			await makeAgentAvailable();
+
+			const latestRoom = await getLivechatRoomInfo(newRoom._id);
+
+			expect(latestRoom).to.have.property('departmentId');
+			expect(latestRoom.departmentId).to.be.equal(forwardToOfflineDepartment._id);
+
+			expect(latestRoom).to.have.property('lastMessage');
+			expect(latestRoom.lastMessage?.t).to.be.equal('livechat_transfer_history');
+			expect(latestRoom.lastMessage?.u?.username).to.be.equal(adminUsername);
+			expect((latestRoom.lastMessage as any)?.transferData?.comment).to.be.equal('test comment');
+			expect((latestRoom.lastMessage as any)?.transferData?.scope).to.be.equal('department');
+			expect((latestRoom.lastMessage as any)?.transferData?.nextDepartment?._id).to.be.equal(forwardToOfflineDepartment._id);
+
+			await updateSetting('Livechat_Routing_Method', 'Manual_Selection');
+			await deleteDepartment(initialDepartment._id);
+			await deleteDepartment(forwardToOfflineDepartment._id);
+		});
+
+		(IS_EE ? it : it.skip)('when manager forward to offline department the inquiry should be set to the queue', async () => {
+			await updateSetting('Livechat_Routing_Method', 'Manual_Selection');
+			const { department: initialDepartment } = await createDepartmentWithAnOnlineAgent();
+			const { department: forwardToOfflineDepartment, agent: offlineAgent } = await createDepartmentWithAnOfflineAgent({
+				allowReceiveForwardOffline: true,
+			});
+
+			const newVisitor = await createVisitor(initialDepartment._id);
+			const newRoom = await createLivechatRoom(newVisitor.token);
+
+			await makeAgentUnavailable(offlineAgent.credentials);
+
+			const manager: IUser = await createUser();
+			const managerCredentials = await login(manager.username, password);
+			await createManager(manager.username);
+
+			await request.post(api('livechat/room.forward')).set(managerCredentials).send({
+				roomId: newRoom._id,
+				departmentId: forwardToOfflineDepartment._id,
+				clientAction: true,
+				comment: 'test comment',
+			});
+
+			await request
+				.get(api(`livechat/queue`))
+				.set(credentials)
+				.query({
+					count: 1,
+				})
+				.expect('Content-Type', 'application/json')
+				.expect(200)
+				.expect((res: Response) => {
+					expect(res.body).to.have.property('success', true);
+					expect(res.body.queue).to.be.an('array');
+					expect(res.body.queue[0].chats).not.to.undefined;
+					expect(res.body).to.have.property('offset');
+					expect(res.body).to.have.property('total');
+					expect(res.body).to.have.property('count');
+				});
+
+			await updateSetting('Livechat_Routing_Method', 'Manual_Selection');
```

Comment:
The default value for this is Auto_Selection, can we put this in an after all hook? So we can reset the default after everything finishes